### PR TITLE
Remove damage to property from risks

### PR DESCRIPTION
**Pull request by cesidio**
This will also change the wording for arson questions.

### CODE REVIEW

**Review by lostie — APPROVED** [review]
LGTM 👍

File: spec/support/fixtures/pdf-text-all-yes-answers.txt [hunk 1]
```diff
@@ -83,53 +83,52 @@ Intimidator/bully                 Yes
 Sex offender                      Yes
   Adult male                      Yes
   Adult female                    Yes
-W1234BY: McTest Testy                                                         Page 2 of 3
```
Comment (dan-palmer) [inline spec/support/fixtures/pdf-text-all-yes-answers.txt:86]: 💡 🔑  if you want to review these pdf changes ignoring the whitespace you can whack `?w=1` to the end of the url. [por exemplo.](https://github.com/ministryofjustice/moving-people-safely/pull/272/files?w=1)

screen shot 2017-04-21 at 15 33 15

Comment (dan-palmer) [reply spec/support/fixtures/pdf-text-all-yes-answers.txt:86]: Not that it makes up for how annoying these tests are.

Comment (cesidio) [reply spec/support/fixtures/pdf-text-all-yes-answers.txt:86]: Thanks!

**Pull request merged**